app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def create
+    @post = current_user.comments.new(comment_params)
```

Comment:
The variable should be `@comment`. Make sure you don't just take it from the notes.